<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QWidget" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="use_password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_login">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_number">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>账户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
